### ISSUE TITLE
[Bug] Talkback cannot access "common myths about private browsing" link from private browsing description

### ISSUE THREAD

**Issue by mcarare** [mozilla-mobile/fenix#5356]
## Steps to reproduce

Launch Fenix with Talkback active.
Switch to private browsing
Access "common myths about private browsing" link

### Expected behavior
The link can receive focus and be accessed 
### Actual behavior
The link cannot receive focus andTalkback users cannot access it

### Device information

* Android device: Any
* Fenix version: 1.0.1938

**Comment by mcarare** [comment]
Similar to #5141

**Comment by softvision-miralobontiu** [comment]
Verified as fixed on the latest Nightly build from PlayStore 190925 (#12681805) with Samsung Galaxy Note 8 (Android 9), Huawei Honor 8 (Android 7), and LG G7 Fit (Android 8.1).
